webdriverio/webdriver/examples/typescript/test/pageobjects/login.page.ts

```diff
@@ -0,0 +1,27 @@
+import Page from './page';
+
+/**
+ * sub page containing specific selectors and methods for a specific page
+ */
+class LoginPage extends Page {
+    /**
+     * The current implementation uses the locators inside of the method.
+     * As the page object expands, you will find that you are reusing the 
+     * locators in multiple places. In that case, you might want to move
+     * them into a private property
+     */
+    login (username: string, password: string):void {
+        $('#username').setValue(username);
```

Comment:
Let's continue our conversation here 😉 

Do you want to keep this method with the selectors in it or do you want to create `get`-ers for them like you already have in `secure.page.ts`?

My suggestion would be to have it more aligned with the POs we have in other projects, meaning having each selector in a `get` and abstract them from the methods

Reply:
@nikolay-advolodkin 

> I just wanted to explain my reasoning for not doing it.

I totally understand, but by making them `private` they won't be abused. You then also provide the "good practice", but up to you. I would go for consistency because you already have it (but then I would even limit the usage of the `flashAlert` and provide a method to use it).
But having said that, let's keep it this way. I'll add a todo on my backlog to add examples for the sample app so everything is using the same app